Dockerfile

```diff
@@ -17,8 +17,9 @@ RUN wget https://apt.llvm.org/llvm-snapshot.gpg.key && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 WORKDIR /usr/local/src
 # Build libbpf and bpftool from the newest stable kernel sources.
+ENV KERNEL_TAG v5.13.1
```

Comment:
Is this something that could be overridden at build? If so, would it make sense to make it an ARG?

Reply:
Good point, ARG can be overriden at build, ENV can't. Will change to ARG then.

Reply:
Done. I also added a possibility to override the tag in Makefile, so you can do:

```
make KERNEL_TAG=v5.12.15
```

or use any tag from https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/